game-core/src/test/java/games/strategy/engine/framework/map/download/ContentDownloaderIntegrationTest.java

```diff
@@ -0,0 +1,29 @@
+package games.strategy.engine.framework.map.download;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.collection.IsEmptyCollection.empty;
+
+import games.strategy.triplea.UrlConstants;
+import games.strategy.triplea.settings.AbstractClientSettingTestCase;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Test;
+import org.triplea.test.common.Integration;
+
+@Integration
+class ContentDownloaderIntegrationTest extends AbstractClientSettingTestCase {
+  @Test
+  void downloadFile() throws Exception {
+    try (CloseableDownloader contentDownloader =
+        new ContentDownloader(URI.create(UrlConstants.DOWNLOAD_WEBSITE))) {
+
+      final List<String> content =
+          IOUtils.readLines(contentDownloader.getStream(), StandardCharsets.UTF_8);
```

Comment:
Warning, opinion:
Not really a fan of commons-io, starting with java 8 I believe the standard provides sufficient API, in this example you could use:
```java
try (var bufferedReader = new BufferedReader(new InputStreamReader(contentDownloader.getStream(), StandardCharsets.UTF_8))) {
  final List<String> content = bufferedReader.lines().collect(Collectors.toList())
}
```
If hamcrest has matchers for streams, then of course there's no need for the collection step

Also when operating with files directly, the Files class has some nicer APIs: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/Files.html